doc/user/content/manage/dbt/development-workflows.md

```diff
@@ -365,17 +365,27 @@ deployment environment to ensure it's safe to [cutover](#cutover-and-cleanup).
    hydrate before you can safely cut over. Use the `run-operation` command
    to invoke the [`deploy_await`](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_await.sql)
    macro, which periodically polls the cluster readiness status, and waits for all
-   objects to have a **lag of less than 1 second** to return successfully.
+   objects to be fully hydrated and have a lag less than the specified tolerance.
 
     ```bash
     dbt run-operation deploy_await
     ```
 
-    The default poll interval is `15` seconds, but can be configured to a
-    different value by passing the `poll_interval=<value>` argument in the macro
-    invocation.
+    By default, `deploy_await` waits for all objects to have a **lag of less than 1 second**
+    before returning successfully. You can customize this behavior with the following arguments:
 
-1. Once `deploy_await` returns successfully, you can manually run tests against
+    - `poll_interval`: The time (in seconds) between each readiness check. Default is 15 seconds.
+    - `lag_tolerance`: The maximum acceptable lag duration. Default is '1s' (1 second).
```

Comment:
```suggestion
    - `lag_threshold`: The maximum lag threshold, which determines when all objects in the environment are considered hydrated and it's safe to perform the cutover step. Default: '1s' (1 second).
```

Reply:
@bobbyiliev, let's s/`lag_tolerance`/`lag_threshold`. I'll push documentation edits in a separate commit, in the meantime.

Reply:
Pushed the changed in [c8f165f846719ad937b1342b7573329f3278846](https://github.com/MaterializeInc/materialize/pull/28832/commits/c8f165f846719ad937b1342b7573329f32788468). Holler if you disagree!

Reply:
Oh awesome! Thank you @morsapaes 🙌 I was just getting to this now.

I like the new `lag_threshold` better!